creation/interfaces-and-networks.md

```diff
@@ -692,3 +687,209 @@ documentation](https://github.com/openshift/sriov-network-operator/blob/master/d
+respective configurations) is defined.
+
+Please refer to the following `SriovNetworkNodePolicy` for an example:
+
+```yaml
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: policy-1
+  namespace: sriov-network-operator
+spec:
+  deviceType: vfio-pci
+  mtu: 1500
+  nicSelector:
+    pfNames:
+    - ens1f0
+  nodeSelector:
+    sriov: "true"
+  numVfs: 8
+  priority: 90
+  resourceName: sriov_net
+```
+
+The policy above will configure the `SR-IOV` device plugin, allowing the
+PF named `ens1f0` to be exposed in the SRIOV capable nodes as a resource named
+`sriov_net`.
+
+It will also render an associated `SriovNetworkNodeState`, which looks like:
+
+```
+apiVersion: v1
+items:
+- apiVersion: sriovnetwork.openshift.io/v1
+  kind: SriovNetworkNodeState
+  metadata:
+    name: sriov-worker
+    namespace: sriov-network-operator
+    ownerReferences:
+    - apiVersion: sriovnetwork.openshift.io/v1
+      blockOwnerDeletion: true
+      controller: true
+      kind: SriovNetworkNodePolicy
+      name: default
+  spec:
+    interfaces:
+    - mtu: 1500
+      name: ens1f0
+      numVfs: 8
+      pciAddress: 0000:65:00.0
+      vfGroups:
+      - deviceType: vfio-pci
+        resourceName: sriov_net
+        vfRange: 0-7
+  status:
+    interfaces:
+    - Vfs:
+      - deviceID: "1016"
+        driver: vfio-pci
+        pciAddress: 0000:65:00.2
+        vendor: 15b3
+        vfID: 0
+      - deviceID: "1016"
+        driver: vfio-pci
+        pciAddress: 0000:65:00.3
+        vendor: 15b3
+        vfID: 1
+      - deviceID: "1016"
+        driver: vfio-pci
+        pciAddress: 0000:65:00.4
+        vendor: 15b3
+        vfID: 2
+      - deviceID: "1016"
+        driver: vfio-pci
+        pciAddress: 0000:65:00.5
+        vendor: 15b3
+        vfID: 3
+      - deviceID: "1016"
+        driver: vfio-pci
+        pciAddress: 0000:65:00.6
+        vendor: 15b3
+        vfID: 4
+      - deviceID: "1016"
+        driver: vfio-pci
+        pciAddress: 0000:65:00.7
+        vendor: 15b3
+        vfID: 5
+      - deviceID: "1016"
+        driver: vfio-pci
+        pciAddress: 0000:65:01.0
+        vendor: 15b3
+        vfID: 6
+      - deviceID: "1016"
+        driver: vfio-pci
+        pciAddress: 0000:65:01.1
+        vendor: 15b3
+        vfID: 7
+      deviceID: "1015"
+      driver: mlx5_core
+      mtu: 1500
+      name: ens1f0
+      numVfs: 8
+      pciAddress: 0000:65:00.0
+      totalvfs: 8
+      vendor: 15b3
+    - deviceID: "1015"
+      driver: mlx5_core
+      mtu: 1500
+      name: ens1f1
+      pciAddress: 0000:65:00.1
+      totalvfs: 8
+      vendor: 15b3
+    syncStatus: Succeeded
+```
+
+#### Start an SR-IOV VM
+
+Once all the SR-IOV components are deployed, it is needed to indicate how to
+configure the SR-IOV network. Refer to the following
+`NetworkAttachmentDefinition` for an example:
+
+    apiVersion: "k8s.cni.cncf.io/v1"
```

Comment:
Could you use the same way to keep code-blocks as for the yamls?

Reply:
Sure I can.